### ISSUE TITLE
[BUG] Unable to publish package

### ISSUE THREAD

**Issue by asos-craigmorten** [nestlandofficial/nest.land#93]
**Describe the bug**

Publishing of package results in the log `[ERROR:publish] Something broke...`.

**To Reproduce**

Experienced locally and in pipeline. See https://github.com/asos-craigmorten/superdeno/blob/main/.github/workflows/publish-egg.yml

Namely:

```bash
deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.1.0/mod.ts
export PATH="~/.deno/bin:$PATH"
eggs link --key ${NEST_LAND_KEY}
eggs publish
```

where `NEST_LAND_KEY` is my API key.

See https://github.com/asos-craigmorten/superdeno/runs/785754213?check_suite_focus=true for example outcome.

**Expected behavior**

The package should be published.

**Screenshots**

--

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04.4 LTS and MacOS 10.14.6
 - nest.land Version [e.g. 0.1.0]: `0.1.0`

**Smartphone (please complete the following information):**

--

**Additional context**

Egg file here --> https://github.com/asos-craigmorten/superdeno/blob/main/egg.json

**Comment by justablob** [comment]
Hey there! Thanks for reporting this. In a few hours we will be releasing an update to the eggs CLI that makes errors more specific, instead of the, quite frankly useless "Something broke..." we had until now. It'll include the HTTP status code. This will allow us to see where exactly the problem is. Would you be fine with waiting for that release, then re-publish and post the error code you get? Thanks.

**Comment by asos-craigmorten** [comment]
Issue resolved in `0.1.3`, published without error - https://nest.land/package/superdeno.

Thanks!
